### ISSUE TITLE
Plural problem

### ISSUE THREAD

**Issue by repli2dev** [Salamek/gettexttranslator#1]
I have encounered weird behaviour of `Gettext::translate`

Usage:

```
{_'%d images were uploaded and inserted.', 1}<br >
{_'%d images were uploaded and inserted.', 2}<br >
{_'%d images were uploaded and inserted.', 5}
```

Template:

```
msgid "%d images were uploaded and inserted."
msgid_plural "%d images were uploaded and inserted."
msgstr[0] ""
msgstr[1] ""
```

Translation:

```
msgid "%d images were uploaded and inserted."
msgid_plural "%d images were uploaded and inserted."
msgstr[0] "%d image was uploaded and inserted."
msgstr[1] "%d images was uploaded and inserted."
```

What I get:

```
1 image was uploaded and inserted.
2 image was uploaded and inserted.
5 image was uploaded and inserted.
```

It seems to me that the translate function is trying to use the argument as a index into translation array instead of clasifying which group it is and then using the group index.

Is that an error, I am using something not supported or the syntax for this usage is weird?

**Comment by repli2dev** [comment]
I believe that I have found a fix... `Gettext::translate` line 325 should be:

```
eval($tmp . '$message_plural = (int) $plural;');
```

(to properly use the code from eval, where the classification of translation group takes place. What do you think?

**Comment by Salamek** [comment]
Fixed in b806b2303009b40898bcac1245799ffff0d416c3

**Comment by repli2dev** [comment]
Thanks 👍
